doc/devices/overview.rst

```diff
@@ -0,0 +1,20 @@
+Overview
+========
+
+Once Pennylane Cirq is installed, the provided Cirq devices can be accessed straight
+away in PennyLane, without the need to import any additional packages.
+
+Devices
+~~~~~~~
+
+Currently, PennyLane-Cirq provides only *one* Cirq device for PennyLane:
```

Comment:
(Minor suggestion) This suggestion is entirely subjective, just so that we are not too hard on ourselves :blush: 
```suggestion
Currently, PennyLane-Cirq provides one Cirq device for PennyLane:
```